### PR TITLE
modules/tectonic/resources/manifests: change console port to 8080

### LINKED ISSUE #847: Console should not run on port 80 if possible
Currently the console listens on port 80 (https://github.com/coreos/tectonic-installer/blob/master/modules/tectonic/resources/manifests/console/deployment.yaml#L42). It is behind a service (https://github.com/coreos/tectonic-installer/blob/master/modules/tectonic/resources/manifests/console/service.yaml) also exposing 80.

Listening on a low port requires running the pod as root or granting it `CAP_NET_BIND_SERVICE`. Would it be possible to run the console pod on a non-privileged port such as 8080 and then just set that `targetPort` in the service?

cc @amrutac @aaronlevy @sym3tri

### PR DESCRIPTION
**Pull request by squat**
fixes #847 

cc @diegs

### CODE REVIEW

**Comment by diegs** [comment]
I just put up https://github.com/coreos/tectonic-installer/pull/854

File: modules/tectonic/resources/manifests/ingress/ingress.yaml [hunk 1]
```diff
@@ -19,8 +19,8 @@ spec:
       - path: /
         backend:
           serviceName: tectonic-console
-          servicePort: 80
+          servicePort: 8080
```
Comment (diegs) [inline modules/tectonic/resources/manifests/ingress/ingress.yaml]: Is this an intrusive change for existing users? I have to write a migration for this in the KVO.

**Comment by sym3tri** [comment]
Let's hold on this for now and address in the _next_ release.

File: modules/tectonic/resources/manifests/ingress/ingress.yaml (hunk 1 above)
Comment (squat) [inline modules/tectonic/resources/manifests/ingress/ingress.yaml]: not an intrusive change. access to the console is handled by the ingress resource, which forwards requests to the console service.

Comment (squat) [inline modules/tectonic/resources/manifests/ingress/ingress.yaml]: closing in favor of #854

**Pull request closed without merging**